pkg/catalog/inbound_traffic_policies_test.go

```diff
@@ -683,6 +671,19 @@ func TestListInboundPoliciesForTrafficSplits(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
+			mockCtrl := gomock.NewController(t)
+			defer mockCtrl.Finish()
```

Comment:
This can be outside the loop as before.

Reply:
As discussed offline, having having these outside the loop was taking the tests to be falky

Reply:
I have updated the PR description and commit to explain the same